### ISSUE TITLE
Missing dependencies

### ISSUE THREAD

**Issue by bmurphey** [igorshubovych/markdownlint-cli#395]
Hello, just wanted to let you know this project's pre-commit hook is breaking due to missing dependencies `rimraf` and later `tsc`.

```
[INFO] Installing environment for https://github.com/igorshubovych/markdownlint-cli.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('/home/me/.cache/pre-commit/repovmzbgxdj/node_env-default/bin/node', '/home/me/.cache/pre-commit/repovmzbgxdj/node_env-default/bin/npm', 'install', '-g', '/home/me/.cache/pre-commit/repovmzbgxdj/markdownlint-cli-0.34.0.tgz')
return code: 127
stdout: (none)
stderr:
    npm ERR! code 127
    npm ERR! git dep preparation failed
    npm ERR! command /home/me/.cache/pre-commit/repovmzbgxdj/node_env-default/bin/node /home/me/.cache/pre-commit/repovmzbgxdj/node_env-default/lib/node_modules/npm/bin/npm-cli.js install --force --cache=/home/me/.npm --prefer-offline=false --prefer-online=false --offline=false --no-progress --no-save --no-audit --include=dev --include=peer --include=optional --no-package-lock-only --no-dry-run
    npm ERR! npm WARN using --force Recommended protections disabled.
    npm ERR! npm ERR! code 127
    npm ERR! npm ERR! path /home/me/.npm/_cacache/tmp/git-cloneqaqeNl
    npm ERR! npm ERR! command failed
    npm ERR! npm ERR! command sh -c npm run compile
    npm ERR! npm ERR! > cliui@8.0.1 precompile
    npm ERR! npm ERR! > rimraf build
    npm ERR! npm ERR! npm WARN using --force Recommended protections disabled.
    npm ERR! npm ERR! sh: 1: rimraf: not found
    npm ERR!
    npm ERR! npm ERR! A complete log of this run can be found in: /home/me/.npm/_logs/2023-05-01T22_05_48_447Z-debug-0.log

    npm ERR! A complete log of this run can be found in: /home/me/.npm/_logs/2023-05-01T22_05_46_666Z-debug-0.log
Check the log at /home/me/.cache/pre-commit/pre-commit.log
```

```
[INFO] Installing environment for https://github.com/igorshubovych/markdownlint-cli.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('/home/me/.cache/pre-commit/repovmzbgxdj/node_env-default/bin/node', '/home/me/.cache/pre-commit/repovmzbgxdj/node_env-default/bin/npm', 'install', '-g', '/home/me/.cache/pre-commit/repovmzbgxdj/markdownlint-cli-0.34.0.tgz')
return code: 127
stdout: (none)
stderr:
    npm ERR! code 127
    npm ERR! git dep preparation failed
    npm ERR! command /home/me/.cache/pre-commit/repovmzbgxdj/node_env-default/bin/node /home/me/.cache/pre-commit/repovmzbgxdj/node_env-default/lib/node_modules/npm/bin/npm-cli.js install --force --cache=/home/me/.npm --prefer-offline=false --prefer-online=false --offline=false --no-progress --no-save --no-audit --include=dev --include=peer --include=optional --no-package-lock-only --no-dry-run
    npm ERR! npm WARN using --force Recommended protections disabled.
    npm ERR! npm ERR! code 127
    npm ERR! npm ERR! path /home/me/.npm/_cacache/tmp/git-clone19Oh3f
    npm ERR! npm ERR! command failed
    npm ERR! npm ERR! command sh -c npm run compile
    npm ERR! npm ERR! > cliui@8.0.1 precompile
    npm ERR! npm ERR! > rimraf build
    npm ERR! npm ERR!
    npm ERR! npm ERR!
    npm ERR! npm ERR! > cliui@8.0.1 compile
    npm ERR! npm ERR! > tsc
    npm ERR! npm ERR! npm WARN using --force Recommended protections disabled.
    npm ERR! npm ERR! sh: 1: tsc: not found
    npm ERR!
    npm ERR! npm ERR! A complete log of this run can be found in: /home/me/.npm/_logs/2023-05-01T22_10_09_155Z-debug-0.log

    npm ERR! A complete log of this run can be found in: /home/me/.npm/_logs/2023-05-01T22_10_06_633Z-debug-0.log
Check the log at /home/me/.cache/pre-commit/pre-commit.log
```

Is that expected?

**Comment by DavidAnson** [comment]
Looks to be a duplicate of #394
